Readme.md

```diff
@@ -42,7 +42,7 @@ const clinical = inputBuilder.buildClinical(
     false,
     false
 );
-const qriskInput = inputBuilder.buildQriskInput(aboutYou, clinical, biometric);
+const qriskInput = inputBuilder.buildQriskInput(aboutYou, clinical, biometric, 0);
```

Comment:
All the other inputs are named variables, might be nice to be consistent.

Reply:
sure